### ISSUE TITLE
Clear logs when big scary button is pressed

### ISSUE THREAD

**Issue by crschardt** [PhotonVision/photonvision#1473]
#1471 adds a button to reset all configurations. This is a request to add a feature to reset the logs at the same time. This can be done by executing `journalctl --rotate --vacuum-time=1s` before restarting photonvision.

**Comment by samfreund** [comment]
![Image](https://github.com/user-attachments/assets/7a3d968e-a87c-4712-bcb7-1cf91906def3)

Not sure if this is what you're referring to here. If so, this can prolly be closed.

**Comment by samfreund** [comment]
oops, wrong button

**Comment by mcm001** [comment]
Nope. Those are (a subset of) current run logs, and do not include[ journalctl logs](https://man7.org/linux/man-pages/man1/journalctl.1.html)
